assemble/bin/accumulo

```diff
@@ -72,13 +72,6 @@ function main() {
   export HADOOP_HOME=$HADOOP_PREFIX
   export HADOOP_HOME_WARN_SUPPRESS=true
 
-  if [ -f "${conf}/jaas.conf" ]; then
-    export JAVA_OPTS=("${JAVA_OPTS[@]}" "-Djava.security.auth.login.config=${conf}/jaas.conf")
```

Comment:
There is no mention of `jaas.conf` anywhere in the documentation or the code.  Users can still add this JVM property to JAVA_OPTS in accumulo-env.sh but I don't see a point in documenting it as it was documented before.